Makefile

```diff
@@ -108,3 +108,8 @@ distance-fixtures:
 	curl -X POST --header "Content-Type:application/json" -d @mapbox/libjava-services/src/test/fixtures/distance_coordinates.json \
 		"https://api.mapbox.com/distances/v1/mapbox/driving?access_token=$(MAPBOX_ACCESS_TOKEN)" \
 		-o mapbox/libjava-services/src/test/fixtures/distance_v1.json
+
+libosrm:
+	rm -rf mapbox/libjava-services/src/main/resources/translations
+	mkdir -p mapbox/libjava-services/src/main/resources/translations
+	cp ../osrm-text-instructions/languages/translations/* mapbox/libjava-services/src/main/resources/translations
```

Comment:
If you check in the JSON files, then the submodule can be something only contributors to this project have to worry about; someone incorporating the library would use the checked-in JSON files but never have to worry about how they got there.